examples/upcloud-cli/main.go

```diff
@@ -87,10 +87,16 @@ func createServer(s *service.Service) error {
 				Tier:    upcloud.StorageTierMaxIOPS,
 			},
 		},
-		IPAddresses: []request.CreateServerIPAddress{
-			{
-				Access: upcloud.IPAddressAccessPrivate,
-				Family: upcloud.IPAddressFamilyIPv4,
+		Networking: &request.CreateServerNetworking{
+			Interfaces: []request.CreateServerInterface{
+				{
+					IPAddresses: []request.CreateServerIPAddress{
+						{
+							Family: upcloud.IPAddressFamilyIPv4,
+						},
+					},
+					Type: upcloud.IPAddressAccessUtility,
```

Comment:
I'd suggest not to use constants from 1.2 here as the interface types in 1.3 are different thing.

Reply:
Hi,

Yes, I've spotted that too. I've added appropriate networking constants in the next batch of changes and switched everything to those.

Would you like me to fix it now or are you happy to wait for the next PR?

Thanks,

Stuart

Reply:
Ok just wanting to check :) Let's get back to this in next PR then.